lib/features/pokemon_overview/pokemon_overview_connector.dart

```diff
@@ -14,6 +15,7 @@ class PokemonOverviewConnector extends StatelessWidget {
   Widget build(BuildContext context) {
     return StoreConnector<AppState, PokemonOverviewVm>(
       vm: () => PokemonOverviewVmFactory(),
+      onInit: (store) async => store.dispatch(GetPokemonListAction()),
```

Comment:
since you are performing an IO request / api request here, what dispatch type do you think is best suited for this case? Would it be just a regular `store.dispatch` or via `store.dispatchAsync`? 🙂

Reply:
I think using async dispatch here would be more suitable since we're accessing api requests

Reply:
because of await, async function can run with parallel with other function and wait for it to complete before proceeding